packages/data-grid/src/table/Table.jsx

```diff
@@ -179,6 +189,31 @@ class Table extends Component {
             headerIndeterminateState={headerIndeterminateState}
           />
           <tbody>
+            {loading && (
+              <tr className="rc-table-cell">
+                <th
+                  className="dg-table-loading-container"
+                  colSpan={columns.length}
+                >
+                  <div
+                    className="dg-table-loading-inner-container"
+                    aria-live="polite"
+                    aria-busy={loading}
```

Comment:
@steveax would this be the correct use of aria-live? Thanks

Reply:
Yep, that looks right to me, thanks!

Reply:
@steveax Thanks for the a11y tips! If those `aria-*` attributes are able to go just on the loading indicator rather than the whole table, I suppose we should move them to the `Loading` component, so we get something like `<div class="rc-loading" aria-live="polite" aria-busy="true"><svg>...</svg></div>` anytime the `Loading` indicator is used. Thoughts?

Reply:
(I ticketed this as [PDS-496](https://tickets.puppetlabs.com/browse/PDS-496).)